pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -66,7 +68,7 @@ def set_start_date_from_period(self, period: int):
         self.start_date = start_dt
 
     @enforce_types
-    def _query_db(self, query: str, scalar=False) -> Union[List[dict], Exception]:
+    def _query_db(self, query: str, scalar=False) -> Union[List[dict], Any]:
```

Comment:
we should fix this typing, it's pandas.dataframe or a scalar (int, float, I think?)